### ISSUE TITLE
Generate usage file with commented-out 0 values

### ISSUE THREAD

**Issue by xeger** [infracost/infracost#930]
When `--sync-usage` adds a default YML to my usage file, it is not known usage and doesn't contribute meaningfully to the cost estimate for my project. It's intended as a placeholder, so the human can put a real value other than 0.

Problem: once the 0 value is in the YML, the resource begins to display as $0 cost. The human may not remember that he needs to add a real usage value!

As a human, I always review the diffs to my usage file and make edits before committing to Git. If `--sync-usage` generated commented-out lines of YAML, it would be easy for me to decide which to uncomment (to establish known usage of 0) and which to provide a real value for. The generated YAML would still provide a guide to me, without polluting my cost estimates with "false $0."

TL;DR, when `--sync-usage` writes a new, defaulted value to the YAML, then instead of this:
```
  aws_autoscaling_group.my_asg:
    instances: 15
    operating_system: linux
    vcpu_count: 2
```

I would like to see this:
```
  aws_autoscaling_group.my_asg:
#    instances: 15
#    operating_system: linux
#    vcpu_count: 2
```

Which will give me the chance to bulk-uncomment the lines if I agree with the defaults, or edit them if I can provide a known, realistic value.

**Comment by alikhajeh1** [comment]
Once we figure-out how to output comments in YAML, it seems like we could also output the comments from the infracost-usage-example.yml into the generated files as without these it's sometimes not clear what the options are (e.g. for pre-defined string keys).

**Comment by hugorut** [comment]
@alikhajeh1 looking to pick this up

**Comment by alikhajeh1** [comment]
@hugorut I pinged you on the community Slack so we coordinate as another contributor is working on related logic
